src/Cake.Common/Tools/NUnit/NUnit3Labels.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Cake.Common.Tools.NUnit
+{
+    /// <summary>
+    /// Represents the possible values for the Labels option
+    /// </summary>
+    public enum NUnit3Labels
+    {
+        /// <summary>
+        /// Does not output labels, the default
+        /// </summary>
+        Off,
```

Comment:
Would prefer this enum value to explicitly be assigned the value of `0` to avoid problems if someone reorder this in the future.